vendors/espressif/boards/esp32s2/aws_demos/config_files/aws_demo_config.h

```diff
@@ -38,6 +38,7 @@
  *          CONFIG_DEVICE_SHADOW_DEMO_ENABLED
  *          CONFIG_DEVICE_DEFENDER_DEMO_ENABLED
  *          CONFIG_OTA_MQTT_UPDATE_DEMO_ENABLED
+ *          CONFIG_OTA_HTTP_UPDATE_DEMO_ENABLED
```

Comment:
Are these actually supported for esp32s2? If not can we mark them as unsupported?

Reply:
We did have the OTA demo enabled for esp32s2: https://github.com/aws/amazon-freertos/blob/7f78ede6884dcf755573eb43731af731fc68374d/vendors/espressif/boards/esp32s2/aws_demos/config_files/aws_demo_config.h#L43

Will need to follow up with the vendor for further clarification and testing.